examples/pipeline_wavernn/transform.py

```diff
@@ -0,0 +1,59 @@
+import librosa
+import numpy as np
```

Comment:
can you add a comment somewhere in module linking the issue to remove the need for librosa? #593

Reply:
I add a comment in [module](https://github.com/pytorch/audio/pull/749/files#diff-f426c1a173746281ffa37227de895169R47)  and link #593 .